app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/ProtvistaPdbValidation.tsx

```diff
@@ -182,22 +198,190 @@ function useGrid() {
+            .text("Per 25");
+
+        svgBar.append("text")
+            .attr("transform", "translate(" + dimensions.width * 0.75 + ", 45)")
+            .style("text-anchor", "middle")
+            .text("Per 75");
+
+        svgBar.append("text")
+            .attr("transform", "translate(" + dimensions.width + ", 45)")
+            .style("text-anchor", "middle")
+            .text("Per 100");
+
+        const percentile25 = svgBar
+            .append("rect")
+            .attr("width", 5)
+            .attr("height", dimensions.height / 6)
+            .attr("x", 0.25 * dimensions.width)
+            .attr("y", 50);
+
+        const percentile75 = svgBar
+            .append("rect")
+            .attr("width", 5)
+            .attr("height", dimensions.height / 6)
+            .attr("x", 0.75 * dimensions.width)
+            .attr("stroke", "black")
+            .attr("stroke-width", 2)
+            .attr("fill", "none")
+            .attr("y", 50);
+
+        const tooltip25 = svgBar.append("g").attr("class", "tooltip25").style("display", "none");
+
+        tooltip25
+            .append("rect")
+            .attr("width", 170)
+            .attr("height", 30)
+            .attr("x", 30)
+            .attr("y", 120)
+            .attr("fill", "lightGray");
+
+        tooltip25
+            .append("line")
+            .attr("stroke", "black")
+            .attr("x1", 100)
+            .attr("x2", 40)
+            .attr("y1", 80)
+            .attr("y2", 120);
+
+        tooltip25
+            .append("line")
+            .attr("stroke", "black")
+            .attr("x1", 105)
+            .attr("x2", 180)
+            .attr("y1", 80)
+            .attr("y2", 120);
+
+        tooltip25
+            .append("text")
+            .text("Quartile-25: " + localResolutionStats.data.metrics[1]?.["quartile 25"] + " Å")
+            .attr("x", "40")
+            .attr("y", "140");
+
+        percentile25.on("mouseover", () => tooltip25.style("display", "block"));
+        percentile25.on("mouseout", () => tooltip25.style("display", "none"));
+
+        const tooltip75 = svgBar.append("g").attr("class", "tooltip75").style("display", "none");
+
+        tooltip75
+            .append("rect")
+            .attr("width", 170)
+            .attr("height", 30)
+            .attr("x", 210)
+            .attr("y", 120)
+            .attr("fill", "lightGray");
+
+        tooltip75
+            .append("line")
+            .attr("stroke", "black")
+            .attr("x1", 300)
+            .attr("x2", 240)
+            .attr("y1", 80)
+            .attr("y2", 120);
+
+        tooltip75
+            .append("line")
+            .attr("stroke", "black")
+            .attr("x1", 305)
+            .attr("x2", 360)
+            .attr("y1", 80)
+            .attr("y2", 120);
+
+        tooltip75
+            .append("text")
+            .text("Quartile-75: " + localResolutionStats.data.metrics[2]?.["quartile 75"] + " Å")
```

Comment:
Accesses looking like array[0], array[1], and so on are discouraged. Better unpack them so they have names and no explicit indexes are used:

```const [name, _ageWithUnderscoreIfUnused, addr = "Default value"] = userFieldsArray```